src/components/ProviderCard.js

```diff
@@ -0,0 +1,284 @@
+import {
+  Fab,
+  Divider,
+  Container,
+  Typography,
+  Box,
+  Card,
+  Stack,
+} from '@mui/material';
+import Grid from '@mui/material/Grid';
+import ReactReadMoreReadLess from 'react-read-more-read-less';
+import { ShortInfo } from './ShortInfo';
+
+import { ArrowForwardIos } from '@mui/icons-material';
+
+import { CheckProfilePic } from '../Utils';
+import { Link } from 'react-router-dom';
+
+export const ProviderCard = ({ provider }) => {
+  const {
+    name,
+    languages,
+    bio,
+    education,
+    title,
+    location,
+    avatarUrl,
+  } = provider || {};
+
+  return (
+    <Container>
+      <Grid container spacing={2} columns={{ xs: 8 }}>
+        <Grid item xs={8} sx={{ mt: 3 }}>
+          <Stack direction="row" alignItems="center" gap={1}>
+            <Typography variant="subtitle1">
+              <Link style={{ textDecoration: 'none' }} to="/">
+                Providers
+              </Link>
+            </Typography>
+            <ArrowForwardIos style={{ fontSize: '12' }} />
+            <Typography variant="subtitle1">{name}</Typography>
+          </Stack>
+        </Grid>
+        <Grid item xs={3} sx={{ mt: 1 }}>
+          <Card sx={{ display: 'flex', height: '350px' }}>
+            <img
+              src={CheckProfilePic(
+                avatarUrl,
+                'https://icon-library.com/images/no-user-image-icon/no-user-image-icon-0.jpg'
+              )}
+              alt={name}
+              width="100%"
+              height="100%"
+            />
+          </Card>
+        </Grid>
+        <Grid item xs={5} sx={{ mt: 1 }}>
+          <Card>
+            <Box sx={{ p: 2, display: 'flex', m: 2 }}>
+              <Stack spacing={0.5}>
+                <Typography variant="h6">
+                  {name}, {title}
+                </Typography>
+
+                <Box style={{ color: '#6E7178' }}>
+                  <ReactReadMoreReadLess
+                    charLimit={180}
+                    readMoreStyle={{
+                      color: '#6277F0',
+                      cursor: 'pointer',
+                    }}
+                    readLessStyle={{
+                      color: '#6277F0',
+                      cursor: 'pointer',
+                    }}
+                    readMoreText={'Read more ▼'}
+                    readLessText={'Read less ▲'}
+                  >
+                    {bio}
+                  </ReactReadMoreReadLess>
+                </Box>
+
+                <Divider />
+
+                <ShortInfo
+                  icon={
+                    <svg
+                      width="32"
+                      height="32"
+                      viewBox="0 0 32 32"
+                      fill="none"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
```

Comment:
Extracting these icon definitions into seperate files will make this file a lot less crowded and easier to read 😄